addon/ng2/blueprints/component/files/__path__/__name__.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component} from 'angular2/core';
+
+@Component({
+  moduleId: __moduleName,
+  selector: '<%= dasherizedModuleName %>',
+  templateUrl: '<%= dasherizedModuleName %>.component.html',
+  styleUrls: ['<%= dasherizedModuleName %>.component.<%= styleExt %>']
+})
+export class <%= classifiedModuleName %>Component {
```

Comment:
@mgechev I agree with that (didn't know it existed) but do you agree with having `ngOnInit` defined by default in a component? I think we should have something other than a constructor in there so devs don't see that as the primary place to put code